README.md

```diff
@@ -204,13 +258,6 @@ This request will match:
 { "fruit": { "apple": "some nonsense" } }
 ```
 
-# On dealing with multiple requests
```

Comment:
Is this no longer relevant?

Reply:
No, this functionality has been removed in v2.  Multiple requests should now be handled using steps from https://github.com/bugsnag/maze-runner/pull/66/files#diff-04c6e90faac2675aa89e2176d2eec7d8R194